dashboard/config/application.rb

```diff
@@ -37,6 +37,12 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, VarnishEnvironment
     config.middleware.insert_after VarnishEnvironment, FilesApi
+
+    if CDO.throttle_data_apis
```

Comment:
Ah.  I guess I'm the misconfigured one then - I still run `redis-server` manually when I need it.